packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -174,11 +174,73 @@ export class VaAlert {
     }
   }
 
+  private addSrOnlyToSlot(): void {
```

Comment:
Is there a reason why this logic can't be part of the render function instead of being called on render?